src/Morse/MorseTable.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { MorseCharacter as Character } from 'puzzle-lib';
+import { Table } from 'react-bootstrap';
+import './MorseTable.css';
+
+class MorseTable extends React.Component {
+  public render() {
+    return (
+      <div className="MorseTable">
+        <Table striped={true} responsive={true}>
+          <thead>
+            <tr>
+              <th>Character</th>
+              <th>Morse</th>
+            </tr>
+          </thead>
+          <tbody>
+            {this.renderRows()}
+          </tbody>
+        </Table>
+      </div>
+    );
+  }
+
+  private renderRows() {
+    const characters = new Character().getPotentialMatches();
+    return characters.map((entry) => (
+      <tr key={entry.display}>
+        <td>{entry.display}</td>
+        <td>{Character.toMorseString(entry.encoding)}</td>

```

Comment:
I think it would look nicer to use the SVG, but I won't block this change on that.

Reply:
I haven't even realized that we were using SVG here.  I'm torn between looking slightly nicer and being able to copy/paste.  I may use the SVG code for this reference table, but when I write an "Encode" tab, I intend to use plain text for that (unless there's some way to make an SVG have copyable alt-text that I don't know about)

Reply:
I mean couldn't you just have a "Copy" button that encoded it into a string?  Select and ctrl-c seems like the least friendly way to do that.

Reply:
I'm also not sure how useful "encode" would be, are there specific scenarios you had in mind?